pack/trors_encounter.json

```diff
@@ -1703,375 +1179,255 @@
-		"threat_fixed": false,
 		"traits": "Hydra. Mutate.",
 		"type_code": "minion"
 	},
 	{
 		"attack": 2,
-		"base_threat_fixed": false,
 		"boost_text": "Place 1 test counter on the main scheme. For each test counter on the main scheme, Zola gets +1 SCH and +1 ATK for this activation.",
-		"card_set_type_name_code": "villain",
 		"code": "04116",
 		"deck_limit": 3,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "encounter",
 		"health": 3,
-		"health_per_hero": false,
-		"hidden": false,
 		"illustrator": "Steve Ellis",
-		"is_unique": false,
 		"name": "Berserk Mutate",
 		"pack_code": "trors",
 		"position": 116,
 		"quantity": 3,
 		"scheme": 0,
 		"set_code": "zola",
 		"set_position": 13,
-		"spoiler": 1,
 		"text": "Quickstrike.",
-		"threat_fixed": false,
 		"traits": "Hydra. Mutate.",
 		"type_code": "minion"
 	},
 	{
-		"base_threat_fixed": false,
 		"boost": 1,
-		"card_set_type_name_code": "villain",
 		"code": "04117",
 		"deck_limit": 3,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "encounter",
-		"health_per_hero": false,
-		"hidden": false,
 		"illustrator": "Patrick McEvoy",
-		"is_unique": false,
 		"name": "Defensive Programming",
 		"pack_code": "trors",
 		"position": 117,
 		"quantity": 3,
 		"set_code": "zola",
 		"set_position": 16,
-		"spoiler": 1,
-		"text": "Attach to the minion with the most remaining hit points without another copy of Defensive Programming attached. If you cannot, this card gains surge.<br>\nAttached minion gets +2 hit points and gains guard.",
-		"threat_fixed": false,
+		"text": "Attach to the minion with the most remaining hit points without another copy of Defensive Programming attached. If you cannot, this card gains surge.\nAttached minion gets +2 hit points and gains guard.",
 		"traits": "Tech.",
 		"type_code": "attachment"
 	},
 	{
-		"base_threat_fixed": false,
 		"boost": 1,
-		"card_set_type_name_code": "villain",
 		"code": "04118",
 		"deck_limit": 2,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "encounter",
-		"health_per_hero": false,
-		"hidden": false,
 		"illustrator": "Andrea Di Vito & Laura Villari",
-		"is_unique": false,
 		"name": "Pain Inhibitors",
 		"pack_code": "trors",
 		"position": 118,
 		"quantity": 2,
 		"set_code": "zola",
 		"set_position": 19,
-		"spoiler": 1,
-		"text": "Attach to the minion with the most remaining hit points without another copy of Pain Inhibitors attached. If you cannot, this card gains surge.<br>\nAttached minion gets +2 hit points and gains retaliate 1.",
-		"threat_fixed": false,
+		"text": "Attach to the minion with the most remaining hit points without another copy of Pain Inhibitors attached. If you cannot, this card gains surge.\nAttached minion gets +2 hit points and gains retaliate 1.",
```

Comment:
General question, aren't encounter cards expected to be tagged as **spoiler** or were there any changes to the app to automatically tagged them based on the faction code or something?

Reply:
There's this in the code that looks like it automatically sets encounter cards as a "spoiler":

```php
if(isset($cardinfo['encounter_code']) && $cardinfo['encounter_code']) {
  $cardinfo['spoiler'] = 1;
}
if(isset($cardinfo['faction_code']) && $cardinfo['faction_code'] == "encounter") {
  $cardinfo['spoiler'] = 1;
}
```

It's in `src/AppBundle/Services/CardsData.php`.

My interpretation (happy to be corrected) is that the "spoiler" field is more for when we're entering leaks and things like that (so I guess Scarlet Witch right now?). I personally don't really care, but if we wanted to do it "right" I think we would make all unreleased cards a "spoiler" and then remove the field when it hits stores. But I guess it hits stores at different times for everyone, so who knows. 🙃

Reply:
That's not a bad idea, sets with unreleased cards could automatically get the spoiler tag, that sounds good to me.